fireplace-swing/src/main/java/com/github/bric3/fireplace/flamegraph/FlameGraphPainter.java

```diff
@@ -240,19 +241,19 @@ private void paintHoveredFrameBorder(Graphics2D g2, Rectangle visibleRect, int f
         }
 
         rect.x = (int) (flameGraphWidth * hoveredFrame.startX) + internalPadding;
-        rect.width = ((int) (flameGraphWidth * hoveredFrame.endX)) - rect.x - internalPadding - frameGapWidth;
+        rect.width = ((int) (flameGraphWidth * hoveredFrame.endX)) - rect.x - internalPadding - frameBorderWidth - (frameGapEnabled ? frameGapWidth : 0);
```

Comment:
```suggestion
        rect.width = ((int) (flameGraphWidth * hoveredFrame.endX)) - rect.x - internalPadding - frameBorderWidth;
```

I believe removing the gap is wrong:
<img width="366" alt="image" src="https://user-images.githubusercontent.com/803621/158577036-083021a3-c81d-4736-b030-d834daf634da.png">